files/build_templates/sonic_debian_extension.j2

```diff
@@ -712,6 +712,7 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get purge
 sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get clean -y
 sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get autoremove -y
 
+cp dockers/docker-fpm-frr/base_image_files/rvtysh dockers/docker-fpm-quagga/base_image_files/rvtysh
```

Comment:
You may create symlink in source code tree. No need to `cp`.

Reply:
Fixed, symlink is much better, thanks.